### ISSUE TITLE
Regression loss functions missing

### ISSUE THREAD

**Issue by umangjpatel** [umangjpatel/dnet#1]
**Feature request description**
Regression algorithms missing. 

**Solution proposed**
Implement loss functions like MSE, RMSE, etc.

**Additional context**
Refer to docs.fast.ai, PyTorch and Keras docs for implementation

**Comment by umangjpatel** [comment]
Regression feature implemented in v0.6.2.2
